docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -132,51 +130,177 @@ The table component consists of the following blocks:
+
+If you want to create an array based on some data from the backend, create an empty array and fill it with items (which corresponds to table rows) inside for loop:
+
+```html+twig
+{% set body_rows = [] %}
+{% for article in pager.currentPageResults %}
+    {# we may render checkbox using form_widget or just put HTML #}
+    {% set col_checkbox %}
+        {{ form_widget(form_remove.articles[article.id]) }}
+    {% endset %}
+    ​
+    {% set col_icon %}
+        <svg class="ibexa-icon ibexa-icon--small">
+            <use xlink:href="{{ ibexa_content_type_icon(article.contentType.identifier) }}"></use>
+        </svg>
+    {% endset %}
+```
+
+### Render hyperlink
 
-    {% block actions %}
-        <a href="#" class="btn btn-icon">
-            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--create">
-                <use xlink:href="{{ ez_icon_path('create') }}"></use>
-            </svg>
+The following example shows how to render both text and hyperlink which redirect to the specified content.
+
+```html+twig
+    {% set col_name %}
+        <a href="{{ path('ibexa.content.view', { contentId: article.contentInfo.id, locationId: article.id }) }}">
+            {{ ibexa_content_name(article.contentInfo) }}
         </a>
-    {% endblock %}
-    
-    {% block thead %}
-        <tr>
-            <th></th>
-            <th>Column A</th>
-            <th>Column B</th>
-            <th>Column C</th>
-        </tr>
-    {% endblock %}
+    {% endset %}
+
+    {% set col_action_btns %}
+        {% if article.userCanEdit %}
+            <a
+                href="#"
+                class="btn ibexa-btn ibexa-btn--ghost ibexa-btn--no-text"
+                title="{{ 'article.list.content.edit'|trans|desc('Edit content') }}"
+            >
+                <svg class="ibexa-icon ibexa-icon--small ibexa-icon--edit">
+                    <use xlink:href="{{ ibexa_icon_path('edit') }}"></use>
+                </svg>
+            </a>
+        {% endif %}
+    {% endset %}
+
+    {% set body_rows = body_rows|merge([{ cols: [
+        { has_checkbox: true, content: col_checkbox },
+        { has_icon: true, content: col_icon },
+        { content: col_name },
+        { content: article.contentType.name },
+        { has_action_btns: true, content: col_action_btns },
+    ]}]) %}
+{% endfor %}
+```
 
-    {% block tbody %}
-        {% for i in 1..10 %}
-            <tr>
-                <td></td>
-                <td>A{{ i }}</td>
-                <td>B{{ i }}</td>
-                <td>C{{ i }}</td>
-            </tr>
-        {% endfor %}
-    {% endblock %}
-{% endembed %}
+### Actions
+
+See the example below to learn how to create an action button which removes the article in the table.
+The table component has to be wrapped into the remove article form.
+
+As in many cases you want a button to be disabled when no item in a table is selected and enabled otherwise, there is a built-in mechanism for this. 
```

Comment:
I'd avoid using "as" in a meaning of "due to a fact", "because of", "while" or "when", as it has so many meanings and can be confusing to less proficient readers.
I would go with "You might want... . There is..."